### PR TITLE
[Font Bakery Dashboard] update family: Solway

### PR DESCRIPTION
**Pull request by vv-monsalve**
[full report](https://fontbakery.graphicore.de/dispatcher/process/fbfdddac-d66a-48d2-881d-66b5f7e0d266)

# Step # 0 Review the Request and Generate the Files Package

**finished: OK**

All tasks completed successfully.

### Task ApproveProcess OK

**@vv-monsalve** confirms the family package.

Report…

### Files
 * `DESCRIPTION.en_us.html` *653 Bytes*
 * `METADATA.pb` *1390 Bytes*
 * `OFL.txt` *4380 Bytes*
 * `Solway-Bold.ttf` *57776 Bytes*
 * `Solway-ExtraBold.ttf` *58164 Bytes*
 * `Solway-Light.ttf` *53176 Bytes*
 * `Solway-Medium.ttf` *57448 Bytes*
 * `Solway-Regular.ttf` *56376 Bytes*

[zip file download](https://fontbakery.graphicore.de/download/persistence/19ac50ba3908f65c4d5b33c29f7a1f79f1a4d4afba6822738762c80262740cd2:2/Solway.zip)

### Metadata
**commit** 598c74737836cee6b8e08b1372277a2bac0cb61a  
**commitDate** 2019-10-28T21:00:25.000Z  
**sourceDetails** 

&emsp;**status** `"OK"`  
&emsp;**name** `"Solway"`  
&emsp;**nameConfirmed** `"Passed"`  
&emsp;**upstream** `"https://github.com/mashavp/Solway"`  
&emsp;**branch** `""`  
&emsp;**fontfilesPrefix** `"fonts/ttf/Solway-"`  
&emsp;**genre** `"Serif"`  
&emsp;**designer** `"Mariya V. Pigoulevskaya"`  
**familyPath** fonts/ttf  
**repository** https://github.com/mashavp/Solway  
**branch** master  
**googleMasterDir** ofl/solway  
**isUpdate** `true`  
**licenseDir** ofl

---

# Step # 1 Quality Assurance

**finished: OK**

All tasks completed successfully.

### Task Fontbakery OK

**@vv-monsalve** Font Bakery looks good.

This is a Font Bakery issue googlefonts/fontbakery#2667

Report…

## [Font Bakery Result](https://fontbakery.graphicore.de/report/4abac5c6-8be7-4262-8ffb-b391e180a818)

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | 🛈 INFO | 🍞 PASS |
|:-----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 5 | 16 | 156 | 36 | 476 |
| 0 % | 0.73 % | 2.32 % | 22.64 % | 5.22 % | 69.09 % |

*created* Mon Oct 28 2019 17:09:26 GMT+0000 (Coordinated Universal Time)<br />
*started* Mon Oct 28 2019 17:09:26 GMT+0000 (Coordinated Universal Time)<br />
*finished* Mon Oct 28 2019 17:09:46 GMT+0000 (Coordinated Universal Time)

### Task Diffenator OK

**@vv-monsalve** Diffenator is skipped.

Font is new in GF

### Task BrowsersDiffsAndPreviews OK

**@vv-monsalve** Browsers diffs and previews are looking good.

Report…

### Preparation Logs
 * `Diffbrowsers version 0.1.6`
 * `Added file "after/DESCRIPTION.en_us.html".`
 * `Added file "after/METADATA.pb".`
 * `Added file "after/OFL.txt".`
 * `Added file "after/Solway-Bold.ttf".`
 * `Added file "after/Solway-ExtraBold.ttf".`
 * `Added file "after/Solway-Light.ttf".`
 * `Added file "after/Solway-Medium.ttf".`
 * `Added file "after/Solway-Regular.ttf".`

### Results
 * browse report: [**Light**](https://fontbakery.graphicore.de/browse/persistence/4e21ef8f9c7d6d606b476472f1299913239c8633cfa2917f4ac08e00f34f6f8f:0/) or download: [zip file](https://fontbakery.graphicore.de/download/persistence/4e21ef8f9c7d6d606b476472f1299913239c8633cfa2917f4ac08e00f34f6f8f:0/Browser%20Diffs_Solway_Light.zip)
 * browse report: [**ExtraBold**](https://fontbakery.graphicore.de/browse/persistence/ebffbd53ba43dd08c6ef672ffc8d9e854afe68ff255ab23666f1f1910edb4f9a:1/) or download: [zip file](https://fontbakery.graphicore.de/download/persistence/ebffbd53ba43dd08c6ef672ffc8d9e854afe68ff255ab23666f1f1910edb4f9a:1/Browser%20Diffs_Solway_ExtraBold.zip)
 * browse report: [**Regular**](https://fontbakery.graphicore.de/browse/persistence/a499a6429978bd43b19dc599ce1582834997180a60620796de5e51507af10e8c:0/) or download: [zip file](https://fontbakery.graphicore.de/download/persistence/a499a6429978bd43b19dc599ce1582834997180a60620796de5e51507af10e8c:0/Browser%20Diffs_Solway_Regular.zip)
 * browse report: [**Bold**](https://fontbakery.graphicore.de/browse/persistence/28bfc23db4589676aaeb0a28233ce43a8076d7cac4bc862e3890e135de38deb1:0/) or download: [zip file](https://fontbakery.graphicore.de/download/persistence/28bfc23db4589676aaeb0a28233ce43a8076d7cac4bc862e3890e135de38deb1:0/Browser%20Diffs_Solway_Bold.zip)
 * browse report: [**Medium**](https://fontbakery.graphicore.de/browse/persistence/af1372cf20bea0bf00779ae6109015a3a96ac69c38fb0e0e85dac9244748e625:0/) or download: [zip file](https://fontbakery.graphicore.de/download/persistence/af1372cf20bea0bf00779ae6109015a3a96ac69c38fb0e0e85dac9244748e625:0/Browser%20Diffs_Solway_Medium.zip)

Download the [full report zip file](https://fontbakery.graphicore.de/download/persistence/collect/4e21ef8f9c7d6d606b476472f1299913239c8633cfa2917f4ac08e00f34f6f8f%3A0%2CLight%3Bebffbd53ba43dd08c6ef672ffc8d9e854afe68ff255ab23666f1f1910edb4f9a%3A1%2CExtraBold%3Ba499a6429978bd43b19dc599ce1582834997180a60620796de5e51507af10e8c%3A0%2CRegular%3B28bfc23db4589676aaeb0a28233ce43a8076d7cac4bc862e3890e135de38deb1%3A0%2CBold%3Baf1372cf20bea0bf00779ae6109015a3a96ac69c38fb0e0e85dac9244748e625%3A0%2CMedium/Browser%20Diffs_Solway.zip).

*started* Tue Oct 29 2019 01:09:52 GMT+0000 (Coordinated Universal Time)<br />
*finished* Tue Oct 29 2019 01:21:59 GMT+0000 (Coordinated Universal Time)

### CODE REVIEW

**Comment by vv-monsalve** [comment]
Updated:

---
[full report](https://fontbakery.graphicore.de/dispatcher/process/a040c5f6-c8c6-42ac-b6ae-3f161227c253)

# Step # 0 Review the Request and Generate the Files Package

**finished: OK**

All tasks completed successfully.

### Task ApproveProcess OK

**@vv-monsalve** confirms the family package.

Report…

## Files Package for *Solway*
### Files
 * `DESCRIPTION.en_us.html` *826 Bytes*
 * `METADATA.pb` *1390 Bytes*
 * `OFL.txt` *4380 Bytes*
 * `Solway-Bold.ttf` *57776 Bytes*
 * `Solway-ExtraBold.ttf` *58164 Bytes*
 * `Solway-Light.ttf` *53176 Bytes*
 * `Solway-Medium.ttf` *57448 Bytes*
 * `Solway-Regular.ttf` *56376 Bytes*


[zip file download](https://fontbakery.graphicore.de/download/persistence/8118a535cfdcb2f21f351df6713fa498d16cfba205d0d10f669c5db869e3fc68:0/Solway.zip)


### Metadata
**commit** 06b46119aa4fef72279bb5dd3d023f0500817982  
**commitDate** 2019-10-31T19:05:12.000Z  
**sourceDetails** 

&emsp;**status** `"OK"`  
&emsp;**name** `"Solway"`  
&emsp;**nameConfirmed** `"Passed"`  
&emsp;**upstream** `"https://github.com/mashavp/Solway"`  
&emsp;**branch** `""`  
&emsp;**fontfilesPrefix** `"fonts/ttf/Solway-"`  
&emsp;**genre** `"Serif"`  
&emsp;**designer** `"Mariya V. Pigoulevskaya"`  
**familyPath** fonts/ttf  
**repository** https://github.com/mashavp/Solway  
**branch** master  
**googleMasterDir** ofl/solway  
**isUpdate** `true`  
**licenseDir** ofl


---

# Step # 1 Quality Assurance

**finished: OK**

All tasks completed successfully.

### Task Fontbakery OK

**@vv-monsalve** Font Bakery looks good.

Font Bakery issue googlefonts/fontbakery#2667

Report…

## [Font Bakery Result](https://fontbakery.graphicore.de/report/821a5039-5909-49c7-a260-038bb90e9698)

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | 🛈 INFO | 🍞 PASS |
|:-----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 5 | 16 | 156 | 36 | 476 |
| 0 % | 0.73 % | 2.32 % | 22.64 % | 5.22 % | 69.09 % |

*created* Thu Oct 31 2019 19:07:25 GMT+0000 (Coordinated Universal Time)<br />
*started* Thu Oct 31 2019 19:07:25 GMT+0000 (Coordinated Universal Time)<br />
*finished* Thu Oct 31 2019 19:07:48 GMT+0000 (Coordinated Universal Time)

### Task Diffenator OK

**@vv-monsalve** Diffenator is skipped.

Is a new font on GF

### Task BrowsersDiffsAndPreviews OK

**@vv-monsalve** Browsers diffs and previews are looking good.

**Comment by m4rc1e** [comment]
Vert metrics differ on each platform.

image
*OS X*

image
*Win*


- Metrics look too tight on Mac.
- Metrics look good on Windows

Please update to the following:

- Enable use_typo_metrics custom parameter in glyphsapp in the source file.
- change vertrical metrics for each master to:

```
typoAscender: 950
typoDescender: -250
typoLineGap: 0
hheaAscender: 950
hheaDescender: -250
hheaLineGap: 0
winAscent: 1000
winDescent: 400
```

**Comment by vv-monsalve** [comment]
Got it! Thanks

**Comment by graphicore** [comment]
I suggest to keep this PR open and re-run the FBD dispatcher when the issue is solved. That way the history of this update stays in the same PR-thread.

**Comment by vv-monsalve** [comment]
Updated:

---
[full report](https://fontbakery.graphicore.de/dispatcher/process/03e4641e-4b0a-4815-a47a-90372572fb0b)

# Step # 0 Review the Request and Generate the Files Package

**finished: OK**

All tasks completed successfully.

### Task ApproveProcess OK

**@vv-monsalve** confirms the family package.

Report…

## Files Package for *Solway*
### Files
 * `DESCRIPTION.en_us.html` *826 Bytes*
 * `METADATA.pb` *1390 Bytes*
 * `OFL.txt` *4380 Bytes*
 * `Solway-Bold.ttf` *57776 Bytes*
 * `Solway-ExtraBold.ttf` *58164 Bytes*
 * `Solway-Light.ttf` *53176 Bytes*
 * `Solway-Medium.ttf` *57448 Bytes*
 * `Solway-Regular.ttf` *56376 Bytes*


[zip file download](https://fontbakery.graphicore.de/download/persistence/f9eaa7fe1d5786156b418ce0285060af9059afe27f9809bebd33990db91ecd71:0/Solway.zip)


### Metadata
**commit** 7707e1d2ab60f923e1a20da47afe13e3ed002348  
**commitDate** 2019-11-04T18:39:56.000Z  
**sourceDetails** 

&emsp;**status** `"OK"`  
&emsp;**name** `"Solway"`  
&emsp;**nameConfirmed** `"Passed"`  
&emsp;**upstream** `"https://github.com/mashavp/Solway"`  
&emsp;**branch** `""`  
&emsp;**fontfilesPrefix** `"fonts/ttf/Solway-"`  
&emsp;**genre** `"Serif"`  
&emsp;**designer** `"Mariya V. Pigoulevskaya"`  
**familyPath** fonts/ttf  
**repository** https://github.com/mashavp/Solway  
**branch** master  
**googleMasterDir** ofl/solway  
**isUpdate** `true`  
**licenseDir** ofl


---

# Step # 1 Quality Assurance

**finished: OK**

All tasks completed successfully.

### Task Fontbakery OK

**@vv-monsalve** Font Bakery looks good.

Bug on FB. Font Bakery issue googlefonts/fontbakery#2667

Report…

## [Font Bakery Result](https://fontbakery.graphicore.de/report/6595f466-ac95-42ad-af8e-1c48241c3a82)

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | 🛈 INFO | 🍞 PASS |
|:-----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 5 | 16 | 156 | 36 | 476 |
| 0 % | 0.73 % | 2.32 % | 22.64 % | 5.22 % | 69.09 % |

*created* Mon Nov 04 2019 18:42:04 GMT+0000 (Coordinated Universal Time)<br />
*started* Mon Nov 04 2019 18:42:04 GMT+0000 (Coordinated Universal Time)<br />
*finished* Mon Nov 04 2019 18:42:24 GMT+0000 (Coordinated Universal Time)

### Task Diffenator OK

**@vv-monsalve** Diffenator is skipped.

Is a new font on GF

### Task BrowsersDiffsAndPreviews OK

**@vv-monsalve** Browsers diffs and previews are looking good.

The existing PR was reopened to include OTF and description last files

Report…

## Browser diffs and previews Result
### Preparation Logs
 * `Diffbrowsers version 0.1.6`
 * `Added file "after/DESCRIPTION.en_us.html".`
 * `Added file "after/METADATA.pb".`
 * `Added file "after/OFL.txt".`
 * `Added file "after/Solway-Bold.ttf".`
 * `Added file "after/Solway-ExtraBold.ttf".`
 * `Added file "after/Solway-Light.ttf".`
 * `Added file "after/Solway-Medium.ttf".`
 * `Added file "after/Solway-Regular.ttf".`

### Results
 * browse report: [**Light**](https://fontbakery.graphicore.de/browse/persistence/0bf86c130e11f55d570a3538da4eecbb335c07b1406d993b6fd5952679738198:0/) or download: [zip file](https://fontbakery.graphicore.de/download/persistence/0bf86c130e11f55d570a3538da4eecbb335c07b1406d993b6fd5952679738198:0/Browser%20Diffs_Solway_Light.zip)
 * browse report: [**ExtraBold**](https://fontbakery.graphicore.de/browse/persistence/32c76161862c9f0e3e9d81bbc9505dcb97b52024cfd6ec3af417b255b757f3b3:0/) or download: [zip file](https://fontbakery.graphicore.de/download/persistence/32c76161862c9f0e3e9d81bbc9505dcb97b52024cfd6ec3af417b255b757f3b3:0/Browser%20Diffs_Solway_ExtraBold.zip)
 * browse report: [**Regular**](https://fontbakery.graphicore.de/browse/persistence/24c804f43a949885128c2473069c92a380fdacf3cb0b4e928117d10f352aca98:0/) or download: [zip file](https://fontbakery.graphicore.de/download/persistence/24c804f43a949885128c2473069c92a380fdacf3cb0b4e928117d10f352aca98:0/Browser%20Diffs_Solway_Regular.zip)
 * browse report: [**Bold**](https://fontbakery.graphicore.de/browse/persistence/4bd9603fd24442116dff8ebb1d0414a901de802a73f752f8dfbd197879909a22:0/) or download: [zip file](https://fontbakery.graphicore.de/download/persistence/4bd9603fd24442116dff8ebb1d0414a901de802a73f752f8dfbd197879909a22:0/Browser%20Diffs_Solway_Bold.zip)
 * browse report: [**Medium**](https://fontbakery.graphicore.de/browse/persistence/30d1b683c9eb7cb635c059638c05e6b29d2d25e4e01541a60731d4e9d99a9353:0/) or download: [zip file](https://fontbakery.graphicore.de/download/persistence/30d1b683c9eb7cb635c059638c05e6b29d2d25e4e01541a60731d4e9d99a9353:0/Browser%20Diffs_Solway_Medium.zip)

Download the [full report zip file](https://fontbakery.graphicore.de/download/persistence/collect/0bf86c130e11f55d570a3538da4eecbb335c07b1406d993b6fd5952679738198%3A0%2CLight%3B32c76161862c9f0e3e9d81bbc9505dcb97b52024cfd6ec3af417b255b757f3b3%3A0%2CExtraBold%3B24c804f43a949885128c2473069c92a380fdacf3cb0b4e928117d10f352aca98%3A0%2CRegular%3B4bd9603fd24442116dff8ebb1d0414a901de802a73f752f8dfbd197879909a22%3A0%2CBold%3B30d1b683c9eb7cb635c059638c05e6b29d2d25e4e01541a60731d4e9d99a9353%3A0%2CMedium/Browser%20Diffs_Solway.zip).

*started* Mon Nov 04 2019 18:43:37 GMT+0000 (Coordinated Universal Time)<br />
*finished* Mon Nov 04 2019 18:55:31 GMT+0000 (Coordinated Universal Time)

**Comment by vv-monsalve** [comment]
PR was reopened and updated with reviewed Vertical metrics .ttf files as well as Description file

**Comment by m4rc1e** [comment]
LGTM.

File: ofl/solway/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
...
-
-<p>To contribute, see <a href="https://github.com/TheNorthernBlock/solway">github.com/TheNorthernBlock/solway</a></p>
+<p>Solway is a playful slab-serif with a charming and understated tone of voice, inspired by the warmth of community and the idyll of rural-living.</p>
+<p>It is a contemporary family defined by the soft corners, ball terminals, rounded slabs that give it a welcoming and friendly appeal. Its monoline structure and minimal stroke contrast aspire to cohesion and stability.</p>
+<p>Available in five weights, it is suitable for a variety of projects. It could find perfect expression within lifestyle brands or nature, travel and foodie blogs.</p>
+<p>Designed by <a href="http://www.metype.co.uk" target="_blank">Mariya V. Pigoulevskaya</a> for <a href="https://thenorthernblock.co.uk" target="_blank">The Northern Block</a>. To colaborate, see <a href="https://github.com/mashavp/Solway" target="_blank">Solway GitHub</a>.</p>
```
Comment (davelab6) [inline ofl/solway/DESCRIPTION.en_us.html]: `colaborate` typo, should be `collaborate`

Comment (vv-monsalve) [reply ofl/solway/DESCRIPTION.en_us.html]: It should be `contribute` actually!
How can I fix it? Should I change it on the repo and restart the PR or it should be another way, given it's already merged.

Comment (graphicore) [reply ofl/solway/DESCRIPTION.en_us.html]: Just fix it and run the dispatcher again.

Comment (vv-monsalve) [reply ofl/solway/DESCRIPTION.en_us.html]: Ok, thanks! Fixed on [https://github.com/google/fonts/pull/2252](https://github.com/google/fonts/pull/2252)

**Pull request merged**